lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -2041,7 +2048,7 @@ def cmd_grep(*args)
     prompt_char = framework.datastore['PromptChar'] || Msf::Ui::Console::Driver::DefaultPromptChar
     mod = active_module
     if mod # if there is an active module, give them the fanciness they have come to expect
-      driver.update_prompt("#{prompt} #{mod.type}(%bld%red#{mod.shortname}%clr) ", prompt_char, true)
+      driver.update_prompt("#{prompt} #{mod.type}(%bld%red#{mod.promptname}%clr) ", prompt_char, true)
```

Comment:
Extra nasty, Framework. Extra.

Reply:
Oops, guess we missed that when we added `promptname`.

Reply:
No worries.